client/src/Component/Footer/Footer.jsx

```diff
@@ -0,0 +1,7 @@
+import "./fotter.css";
+
+const Footer = () => {
+  return <div>footer</div>;
```

Comment:
Is this any use? 🙈

Reply:
I had plan to add footer at the start of the projetct , I've  just deleted